### ISSUE TITLE
Hint about position when writing function args

### ISSUE THREAD

**Issue by bilekt3** [cursiveclojure/cursive#1121]
It will be handy when editor will highlight position of arguments (CTRL+P). Especially in Java interoperability where long function signatures are not rare.

For example:

Clojure
![clojure](https://cloud.githubusercontent.com/assets/381740/10789716/58062f48-7d7f-11e5-8133-3611cf340216.png)

Java
![java](https://cloud.githubusercontent.com/assets/381740/10789755/915edc18-7d7f-11e5-85c9-00dea275d10d.png)

**Comment by cursiveclojure** [comment]
Yes, this would be possible (and desirable) for interop. It's tricky with a lot of forms in Clojure since basic forms like `defn` often have arglists like `name & rest`. But it should be possible to do something interesting with the other cases, though.

**Comment by mtruyens** [comment]
+1

**Comment by bilekt3** [comment]
After some times I would like to vote for this feature because I have to write some Java and PHP in IntelliJ IDEA and this position indicator is very helpful. name & rest is not issue for me. At leas I will se where I am in function args.
